spatial-tracking-explainer.md

```diff
@@ -228,18 +233,20 @@ For viewer poses, it's common in VR experiences to let the world drag along with
 
 For input source poses, motion controllers may have their own inertial tracking that can continue updating orientation while positional tracking is lost. If using a controller primarily to target distant objects, developers may simply ignore `emulatedPosition` and continue to point using each frame's updated target ray as the user rotates the controller. If using a controller to do fine operations, such as painting at the controller's tip, developers may instead choose to stop painting when `emulatedPosition` becomes false, ensuring that only high-quality paint strokes are drawn.
 
-#### Rays
-An `XRRay` object includes both an `origin` and `direction`, both given as `DOMPointReadOnly`s. The `origin` represents a 3D coordinate in space with a `w` component that must be 1, and the `direction` represents a normalized 3D directional vector with a `w` component that must be 0. The `XRRay` also defines a `matrix` which represents the transform from a ray originating at `[0, 0, 0]` and extending down the negative Z axis to the ray described by the `XRRay`'s `origin` and `direction`. This is useful for positioning graphical representations of the ray.
+#### Tracking recovery
+As discussed above, during tracking loss, the viewer's pose will remain at their last-known position relative to each reference space, while continuing to incorporate orientation updates and any slight adjustments due to neck modeling. When the viewer then recovers positional tracking, developers may observe an instantaneous jump of the viewer pose. If the application continues to render during this time, the world will appear to drag along as the viewer moves around.
```

Comment:
should this also mention input sources? It currently only covers tracking being lost for viewer spaces.

Reply:
Should also redundantly mention that emulatedPosition is false, ideally

Reply:
Thanks!  Added coverage of tracking recovery for input source poses.